control-plane/subcommand/sync-catalog/command.go

```diff
@@ -251,6 +252,13 @@ func (c *Command) Run(args []string) int {
 	}
 	c.ready = true
 
+	if !c.flagToConsul && !c.flagToK8S {
+		if err := c.removeAllK8SServicesFromConsul(consulConfig); err != nil {
```

Comment:
can we log the error here?

Reply:
Yup, adjusted the place to log the error